change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -2060,8 +2123,26 @@ PROCEDURE VALIDATECONFIG
        PARAMVALIDATIONERROR=TRUE
        PARAMVALIDATIONERRORCHR="No email detail lines specified for member email."
       END
-
-     ELSE IF PARAMEMAILFROM<>MEMBEREMAIL THEN
+    END
+[* If allow in memo mode is enabled, check for email detail line. If none specified, add default lines.
+*]     
+   IF PARAMALLOWINMEMOMODE=TRUE AND
+      MMEBEMAILLINECOUNT=0 AND
+      MMEBEMAILLINECOUNT+3<EMAILLINESMAX THEN
+    DO [Add default memo email body lines if memo is allowed and configuration lines are blank and fit the array.]
+     PARAMMEMOMODEEMAILTEXT(1)="Nightly processing is underway. Your financial institution address information has been updated"
+     PARAMMEMOMODEEMAILTEXT(2)="and will be reflected the next business day."
+     PARAMMEMOMODEEMAILTEXT(3)="Please contact us immediately if you did not make these changes: (xxx) xxx-xxxx"
+     MMEBEMAILLINECOUNT=MMEBEMAILLINECOUNT+3
```

Comment:
Why add 3 to the line count and then hard set it to 3 again? Since this is only called when you're adding three default message lines, then I think the MMEBEMAILLINECOUNT=3 would be sufficient by itself - no?

Reply:
Thanks; I've made the change and it will be in the next commit.